scipy/stats/_distn_infrastructure.py

```diff
@@ -3470,7 +3470,6 @@ def fun(x):
         # might be problems(?) with correct self.a, self.b at this stage maybe
```

Comment:
nit: does the docstring having a typo?

```
 lb, ub : int, optional
            Lower and upper bound for the summation, default is set to the
            support of the distribution, inclusive (``ul <= k <= ub``).	            
```
should that be (``**lb** <= k <= ub``).

Reply:
Nice catch. The typo seems to have been there forever (tm). Fixed in the last commit.